specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,498 @@
+        <Setter Property="BorderBrush" Value="{ThemeResource CalendarViewBorderBrush}" />
+        <Setter Property="DayItemFontFamily" Value="XamlAutoFontFamily" />
+        <Setter Property="DayItemFontSize" Value="{ThemeResource CalendarViewDayItemFontSize}" />
+        <Setter Property="DayItemMargin" Value="{ThemeResource CalendarViewDayItemMargin}" /> *
+        <Setter Property="FirstOfMonthLabelFontFamily" Value="XamlAutoFontFamily" />
+        <Setter Property="FirstOfMonthLabelFontSize" Value="{ThemeResource CalendarViewFirstOfMonthLabelFontSize}" />
+        <Setter Property="FirstOfMonthLabelMargin" Value="{ThemeResource CalendarViewFirstOfMonthLabelMargin}" /> *
+        <Setter Property="MonthYearItemFontFamily" Value="XamlAutoFontFamily" />
+        <Setter Property="MonthYearItemFontSize" Value="{ThemeResource CalendarViewMonthYearItemFontSize}" />
+        <Setter Property="MonthYearItemMargin" Value="{ThemeResource CalendarViewMonthYearItemMargin}" /> *
+        <Setter Property="FirstOfYearDecadeLabelFontFamily" Value="XamlAutoFontFamily" />
+        <Setter Property="FirstOfYearDecadeLabelFontSize" Value="{ThemeResource CalendarViewFirstOfYearDecadeLabelFontSize}" />
+        <Setter Property="FirstOfYearDecadeLabelMargin" Value="{ThemeResource CalendarViewFirstOfYearDecadeLabelMargin}" /> *
+        <Setter Property="CalendarItemBorderThickness" Value="1" />
+        <Setter Property="BorderThickness" Value="1" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="VerticalAlignment" Value="Center" />
+        <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+        <Setter Property="VerticalContentAlignment" Value="Stretch" />
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+        <Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+        <Setter Property="Template">
+          ...
+        </Setter>
+    </Style>
+</ResourceDictionary>
+```
+
+The new properties appear with a * on their right.
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+The Windows OS code switches to the new rendering, capable of showing calendar items with rounded corners, when there is a boolean theme resource named CalendarViewBaseItemRoundedChromeEnabled set to True.
+WinUI will set that resource to True for applications that use 'Version2' for ControlsResourcesVersion (<controls:XamlControlsResources ControlsResourcesVersion="Version2"/>).
```

Comment:
It's called out at the top that this is a WUX feature, not MUXC, but it's still controlled by the XamlControlsResources?

Reply:
Yes. Like for the new ListViewItemPresenter rounded corner rendering, the feature is turned off by default because the OS' generic.xaml sets the CalendarViewBaseItemRoundedChromeEnabled resource to False.

The feature is turned on in WinUI 2.6 when the app is built for October Refresh by setting CalendarViewBaseItemRoundedChromeEnabled to True.